src/lib/mina_lib/coda_subscriptions.ml

```diff
@@ -132,40 +180,27 @@ let create ~logger ~constraint_constants ~wallets ~new_blocks
                 in
                 Precomputed.to_yojson precomputed_block )
            in
-           if upload_blocks_to_gcloud then (
-             [%log info] "log" ;
+           (* Upload precomputed blocks to gcloud *)
+           ( if upload_blocks_to_gcloud then
              let json = Yojson.Safe.to_string (Lazy.force precomputed_block) in
-             let network =
-               match Core.Sys.getenv "NETWORK_NAME" with
-               | Some network ->
-                   Some network
-               | _ ->
-                   [%log warn]
-                     "NETWORK_NAME environment variable not set. Must be set \
-                      to use upload_blocks_to_gcloud" ;
-                   None
-             in
-             let bucket =
-               match Core.Sys.getenv "GCLOUD_BLOCK_UPLOAD_BUCKET" with
-               | Some bucket ->
-                   Some bucket
-               | _ ->
-                   [%log warn]
-                     "GCLOUD_BLOCK_UPLOAD_BUCKET environment variable not set. \
-                      Must be set to use upload_blocks_to_gcloud" ;
-                   None
-             in
-             match (gcloud_keyfile, network, bucket) with
-             | Some _, Some network, Some bucket ->
+             match (gcloud_keyfile, gcloud_bucket) with
+             | Some _, Some bucket ->
                  let hash_string = State_hash.to_base58_check hash in
                  let height =
-                   Mina_block.Validated.forget new_block
```

Comment:
Why the indentation changed here?

Reply:
Forgot to hit it with reformat. My apologies!